doctests/README.md

```diff
@@ -0,0 +1,23 @@
+# Command examples for redis.io
+
+These examples appear on the [Redis documentation](https://redis.io) site as part of the tabbed examples interface.
+
+## How to add examples
+
+- Create a Go test file with a meaningful name in the current folder. 
+- Create a single method prefixed with `Example` and write your test in it.
+- Find out what's the example id for the example you're creating and add it as the first line of the file `// EXAMPLE: set_and_get`. See more info below under the [Example](#example) section.
+- We're using the [Testable Examples](https://go.dev/blog/examples) feature of Go to test the desired output has been written to stdout.
+- To run the tests start a Redis server locally on port 6379 and run `go test`. (This will be improved and aligned with the way we run the main test suite).
```

Comment:
Start a Redis server locally, on port 6379 and run `go test`.

Reply:
I might add - this is a duplication, since how to add and how to test contain similar content. Reduce?